[CI] ProactiveStorageIT testScaleUp failing

#93524 introduces a new place where we obtain a `FileStore`:

https://github.com/elastic/elasticsearch/blob/30999fe84c0115ad9933f166882c0c1ef3eb201b/server/src/main/java/org/elasticsearch/index/engine/InternalEngine.java#L236

This trips the following assertion in `DiskUsageIntegTestCase` (but not on Linux):

https://github.com/elastic/elasticsearch/blob/00f4755be372ba423b841ce300d2116623d16b2c/test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java#L229

**Build scan:**
https://gradle-enterprise.elastic.co/s/qc7vrnoboxjgs/tests/:x-pack:plugin:autoscaling:internalClusterTest/org.elasticsearch.xpack.autoscaling.storage.ProactiveStorageIT/testScaleUp

**Reproduction line:**
```
gradlew ':x-pack:plugin:autoscaling:internalClusterTest' --tests "org.elasticsearch.xpack.autoscaling.storage.ProactiveStorageIT.testScaleUp" -Dtests.seed=4F1C800928F2CC3A -Dtests.locale=mk-MK -Dtests.timezone=Asia/Almaty -Druntime.java=20
```

**Applicable branches:**
main

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.autoscaling.storage.ProactiveStorageIT&tests.test=testScaleUp

**Failure excerpt:**
```
java.lang.AssertionError: 
Expected: an empty iterable
     but: [<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][wlHCNocBi_AlOUzA1G-g], source[{"@timestamp":"4947-11-11T07:26:37.482Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][sFHCNocBi_AlOUzA1G-d], source[{"@timestamp":"2411-01-24T03:53:39.500Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][rFHCNocBi_AlOUzA1G-R], source[{"@timestamp":"3785-07-08T20:08:04.600Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][vlHCNocBi_AlOUzA1G-f], source[{"@timestamp":"5355-09-18T01:44:26.092Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][uFHCNocBi_AlOUzA1G-f], source[{"@timestamp":"5725-07-24T12:26:05.432Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][rVHCNocBi_AlOUzA1G-R], source[{"@timestamp":"3996-04-20T21:46:18.643Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][wVHCNocBi_AlOUzA1G-g], source[{"@timestamp":"2174-01-10T15:40:46.982Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][vVHCNocBi_AlOUzA1G-f], source[{"@timestamp":"7488-04-04T23:06:26.223Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][tlHCNocBi_AlOUzA1G-e], source[{"@timestamp":"7188-07-17T10:33:59.897Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][wFHCNocBi_AlOUzA1G-g], source[{"@timestamp":"6251-07-21T02:58:20.035Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][xFHCNocBi_AlOUzA1G-g], source[{"@timestamp":"6854-05-25T02:18:32.444Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][u1HCNocBi_AlOUzA1G-f], source[{"@timestamp":"4866-07-10T18:22:00.971Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][yFHCNocBi_AlOUzA1G-g], source[{"@timestamp":"4368-06-26T16:29:33.516Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][s1HCNocBi_AlOUzA1G-e], source[{"@timestamp":"8555-07-27T19:57:18.923Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][x1HCNocBi_AlOUzA1G-g], source[{"@timestamp":"2390-01-14T19:30:52.452Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][uVHCNocBi_AlOUzA1G-f], source[{"@timestamp":"3911-06-02T22:31:29.539Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][1lHCNocBi_AlOUzA1G-i], source[{"@timestamp":"7105-01-02T13:45:46.424Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][tVHCNocBi_AlOUzA1G-e], source[{"@timestamp":"9197-07-15T04:40:21.073Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][v1HCNocBi_AlOUzA1G-f], source[{"@timestamp":"9431-03-29T17:53:53.709Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][w1HCNocBi_AlOUzA1G-g], source[{"@timestamp":"4147-04-12T01:48:33.395Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][ulHCNocBi_AlOUzA1G-f], source[{"@timestamp":"4248-07-14T10:08:33.679Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][sVHCNocBi_AlOUzA1G-e], source[{"@timestamp":"7885-06-21T20:55:28.359Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][xVHCNocBi_AlOUzA1G-g], source[{"@timestamp":"4821-09-10T01:58:31.274Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][r1HCNocBi_AlOUzA1G-d], source[{"@timestamp":"7879-05-09T01:38:52.930Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][tFHCNocBi_AlOUzA1G-e], source[{"@timestamp":"3540-06-16T03:42:07.566Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][t1HCNocBi_AlOUzA1G-f], source[{"@timestamp":"4158-09-21T11:18:35.574Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][vFHCNocBi_AlOUzA1G-f], source[{"@timestamp":"2122-05-14T06:32:15.034Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][slHCNocBi_AlOUzA1G-e], source[{"@timestamp":"8477-05-04T23:27:14.568Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][2VHCNocBi_AlOUzA1G-i], source[{"@timestamp":"2442-11-12T12:53:15.433Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][zFHCNocBi_AlOUzA1G-h], source[{"@timestamp":"2492-03-31T11:09:24.453Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][xlHCNocBi_AlOUzA1G-g], source[{"@timestamp":"3326-03-30T12:10:35.273Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][y1HCNocBi_AlOUzA1G-h], source[{"@timestamp":"8106-12-15T05:33:12.299Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][z1HCNocBi_AlOUzA1G-h], source[{"@timestamp":"5631-04-18T11:27:20.434Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][q1HCNocBi_AlOUzA1G-R], source[{"@timestamp":"9631-05-01T14:19:35.498Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][rlHCNocBi_AlOUzA1G-d], source[{"@timestamp":"7986-12-19T12:56:06.300Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][31HCNocBi_AlOUzA1G-j], source[{"@timestamp":"4157-03-08T20:43:01.892Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][zlHCNocBi_AlOUzA1G-h], source[{"@timestamp":"3730-08-03T01:27:38.885Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][0FHCNocBi_AlOUzA1G-h], source[{"@timestamp":"6595-01-31T19:28:45.820Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][01HCNocBi_AlOUzA1G-i], source[{"@timestamp":"3942-01-06T05:12:48.119Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][yVHCNocBi_AlOUzA1G-h], source[{"@timestamp":"4994-01-30T03:08:43.534Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][6FHCNocBi_AlOUzA1G-k], source[{"@timestamp":"8674-02-28T07:45:48.271Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][5VHCNocBi_AlOUzA1G-k], source[{"@timestamp":"6244-08-29T16:27:49.162Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][61HCNocBi_AlOUzA1G-l], source[{"@timestamp":"4503-06-23T16:59:45.565Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][11HCNocBi_AlOUzA1G-i], source[{"@timestamp":"5521-03-30T13:38:21.735Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][zVHCNocBi_AlOUzA1G-h], source[{"@timestamp":"4198-07-22T00:27:08.001Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][1FHCNocBi_AlOUzA1G-i], source[{"@timestamp":"5476-06-13T03:03:05.114Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][1VHCNocBi_AlOUzA1G-i], source[{"@timestamp":"8710-11-19T08:43:10.012Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][0VHCNocBi_AlOUzA1G-h], source[{"@timestamp":"8763-09-28T17:57:39.181Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][3FHCNocBi_AlOUzA1G-j], source[{"@timestamp":"9132-07-29T02:23:24.666Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][ylHCNocBi_AlOUzA1G-h], source[{"@timestamp":"8705-06-18T00:22:05.961Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][4lHCNocBi_AlOUzA1G-k], source[{"@timestamp":"8801-09-16T01:15:15.728Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][2FHCNocBi_AlOUzA1G-i], source[{"@timestamp":"7631-11-02T05:59:55.299Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][7lHCNocBi_AlOUzA1G-m], source[{"@timestamp":"9266-06-11T01:51:28.220Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][7VHCNocBi_AlOUzA1G-l], source[{"@timestamp":"4097-10-15T18:53:11.933Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][7FHCNocBi_AlOUzA1G-l], source[{"@timestamp":"7593-09-24T14:17:05.030Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][0lHCNocBi_AlOUzA1G-i], source[{"@timestamp":"7508-04-21T04:34:03.969Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][4VHCNocBi_AlOUzA1G-j], source[{"@timestamp":"8841-10-26T11:23:40.733Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][3VHCNocBi_AlOUzA1G-j], source[{"@timestamp":"7736-01-28T03:14:29.700Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][41HCNocBi_AlOUzA1G-k], source[{"@timestamp":"7017-10-04T02:34:44.470Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][21HCNocBi_AlOUzA1G-j], source[{"@timestamp":"6898-03-05T03:15:06.349Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][6lHCNocBi_AlOUzA1G-k], source[{"@timestamp":"7362-05-29T22:13:07.210Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][3lHCNocBi_AlOUzA1G-j], source[{"@timestamp":"4043-07-24T19:28:10.556Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][5lHCNocBi_AlOUzA1G-k], source[{"@timestamp":"3640-01-22T11:23:37.202Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][4FHCNocBi_AlOUzA1G-j], source[{"@timestamp":"7876-07-16T06:25:08.816Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][51HCNocBi_AlOUzA1G-k], source[{"@timestamp":"8506-08-08T15:52:14.739Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][6VHCNocBi_AlOUzA1G-k], source[{"@timestamp":"2136-11-06T03:47:35.268Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][5FHCNocBi_AlOUzA1G-k], source[{"@timestamp":"4378-11-28T00:10:49.935Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][2lHCNocBi_AlOUzA1G-j], source[{"@timestamp":"4464-11-13T12:36:49.540Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][8lHCNocBi_AlOUzA1G-n], source[{"@timestamp":"2123-03-09T01:29:00.826Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][8VHCNocBi_AlOUzA1G-n], source[{"@timestamp":"2495-10-05T22:08:15.560Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][8FHCNocBi_AlOUzA1G-m], source[{"@timestamp":"5667-01-09T22:57:36.741Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][-VHCNocBi_AlOUzA1G-p], source[{"@timestamp":"4862-12-29T02:30:52.362Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][9FHCNocBi_AlOUzA1G-o], source[{"@timestamp":"9371-03-16T18:55:33.864Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][9lHCNocBi_AlOUzA1G-o], source[{"@timestamp":"7856-11-21T23:55:45.840Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][71HCNocBi_AlOUzA1G-m], source[{"@timestamp":"4564-01-08T23:35:19.003Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][81HCNocBi_AlOUzA1G-n], source[{"@timestamp":"9811-03-06T16:40:25.333Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][91HCNocBi_AlOUzA1G-p], source[{"@timestamp":"6414-08-27T20:43:49.603Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][9VHCNocBi_AlOUzA1G-o], source[{"@timestamp":"3518-08-18T19:17:26.042Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][-FHCNocBi_AlOUzA1G-p], source[{"@timestamp":"3124-04-28T09:47:46.557Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][BlHCNocBi_AlOUzA1HCu], source[{"@timestamp":"9623-07-25T12:28:42.280Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][BVHCNocBi_AlOUzA1HCu], source[{"@timestamp":"6817-10-10T02:41:39.010Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][BFHCNocBi_AlOUzA1HCt], source[{"@timestamp":"2248-05-15T09:07:51.060Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][A1HCNocBi_AlOUzA1HCt], source[{"@timestamp":"6380-01-31T12:59:20.749Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][-1HCNocBi_AlOUzA1G-q], source[{"@timestamp":"3944-07-23T21:55:45.752Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][_lHCNocBi_AlOUzA1G-r], source[{"@timestamp":"8008-04-03T13:35:52.430Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][_FHCNocBi_AlOUzA1G-q], source[{"@timestamp":"8873-05-21T10:15:32.856Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][-lHCNocBi_AlOUzA1G-q], source[{"@timestamp":"4548-08-06T17:27:54.810Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][AFHCNocBi_AlOUzA1HCs], source[{"@timestamp":"2814-03-13T00:17:44.988Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][_1HCNocBi_AlOUzA1G-r], source[{"@timestamp":"6681-10-24T03:59:18.533Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][AVHCNocBi_AlOUzA1HCs], source[{"@timestamp":"8362-09-08T03:36:50.913Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][_VHCNocBi_AlOUzA1G-r], source[{"@timestamp":"6039-11-04T05:02:33.780Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][AlHCNocBi_AlOUzA1HCt], source[{"@timestamp":"3070-02-12T17:39:55.885Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][C1HCNocBi_AlOUzA1HCx], source[{"@timestamp":"8077-05-28T06:45:57.315Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][CFHCNocBi_AlOUzA1HCv], source[{"@timestamp":"7449-09-14T01:21:33.157Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][ClHCNocBi_AlOUzA1HCw], source[{"@timestamp":"3567-06-09T20:02:13.564Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][DVHCNocBi_AlOUzA1HCx], source[{"@timestamp":"8996-06-09T23:38:33.363Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][DFHCNocBi_AlOUzA1HCx], source[{"@timestamp":"4374-11-20T11:30:29.663Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][B1HCNocBi_AlOUzA1HCu], source[{"@timestamp":"3595-09-27T00:28:42.991Z"}]}]]>,<org.elasticsearch.action.UnavailableShardsException: [.ds-jxwpwjrlfg-2023.03.31-000001][0] primary shard is not active Timeout: [1m], request: [BulkShardRequest [[.ds-jxwpwjrlfg-2023.03.31-000001][0]] containing [index {[jxwpwjrlfg][CVHCNocBi_AlOUzA1HCw], source[{"@timestamp":"4747-10-17T05:09:22.182Z"}]}]]>]

  at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
  at org.junit.Assert.assertThat(Assert.java:956)
  at org.junit.Assert.assertThat(Assert.java:923)
  at org.elasticsearch.test.ESIntegTestCase.indexRandom(ESIntegTestCase.java:1660)
  at org.elasticsearch.test.ESIntegTestCase.indexRandom(ESIntegTestCase.java:1573)
  at org.elasticsearch.test.ESIntegTestCase.indexRandom(ESIntegTestCase.java:1538)
  at org.elasticsearch.xpack.autoscaling.storage.ProactiveStorageIT.testScaleUp(ProactiveStorageIT.java:51)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1623)

```

--- a/server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitorIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitorIT.java
@@ -8,7 +8,6 @@
 
 package org.elasticsearch.cluster.routing.allocation;
 
-import org.apache.lucene.tests.util.LuceneTestCase;
 import org.elasticsearch.cluster.DiskUsageIntegTestCase;
 import org.elasticsearch.cluster.metadata.IndexMetadata;
 import org.elasticsearch.cluster.routing.ShardRouting;
@@ -28,7 +27,6 @@ import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcke
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertBlocked;
 import static org.hamcrest.Matchers.equalTo;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/94935")
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class DiskThresholdMonitorIT extends DiskUsageIntegTestCase {
 

--- a/server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDeciderIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDeciderIT.java
@@ -8,7 +8,6 @@
 
 package org.elasticsearch.cluster.routing.allocation.decider;
 
-import org.apache.lucene.tests.util.LuceneTestCase;
 import org.elasticsearch.action.admin.cluster.snapshots.create.CreateSnapshotResponse;
 import org.elasticsearch.action.admin.cluster.snapshots.restore.RestoreSnapshotResponse;
 import org.elasticsearch.action.admin.indices.stats.ShardStats;
@@ -54,7 +53,6 @@ import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/94935")
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class DiskThresholdDeciderIT extends DiskUsageIntegTestCase {
 

--- a/test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java
+++ b/test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java
@@ -224,9 +224,9 @@ public class DiskUsageIntegTestCase extends ESIntegTestCase {
                 return fileStore;
             }
 
-            // On Linux, and only Linux, Lucene obtains a filestore for the index in order to determine whether it's on a spinning disk or
-            // not so it can configure the merge scheduler accordingly
-            assertTrue(path + " not tracked and not on Linux", Constants.LINUX);
+            // We check the total size available for translog in InternalEngine constructor and we allow that here,
+            // expecting to match a unique root path.
+            assertTrue(path + " not tracked and not translog", path.getFileName().toString().equals("translog"));
             final Set<Path> containingPaths = trackedPaths.keySet().stream().filter(path::startsWith).collect(Collectors.toSet());
             assertThat(path + " not contained in a unique tracked path", containingPaths, hasSize(1));
             return trackedPaths.get(containingPaths.iterator().next());

--- a/test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java
+++ b/test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java
@@ -11,7 +11,6 @@ package org.elasticsearch.cluster;
 import org.apache.lucene.tests.mockfile.FilterFileStore;
 import org.apache.lucene.tests.mockfile.FilterFileSystemProvider;
 import org.apache.lucene.tests.mockfile.FilterPath;
-import org.apache.lucene.util.Constants;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.core.IOUtils;
 import org.elasticsearch.core.PathUtils;

--- a/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ProactiveStorageIT.java
+++ b/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ProactiveStorageIT.java
@@ -7,7 +7,6 @@
 
 package org.elasticsearch.xpack.autoscaling.storage;
 
-import org.apache.lucene.tests.util.LuceneTestCase;
 import org.elasticsearch.action.admin.indices.rollover.RolloverRequest;
 import org.elasticsearch.action.admin.indices.stats.IndicesStatsResponse;
 import org.elasticsearch.action.admin.indices.template.put.PutComposableIndexTemplateAction;

--- a/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ProactiveStorageIT.java
+++ b/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ProactiveStorageIT.java
@@ -36,7 +36,6 @@ import java.util.stream.IntStream;
 
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/94935")
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class ProactiveStorageIT extends AutoscalingStorageIntegTestCase {
 

--- a/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageIT.java
+++ b/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageIT.java
@@ -7,7 +7,6 @@
 
 package org.elasticsearch.xpack.autoscaling.storage;
 
-import org.apache.lucene.tests.util.LuceneTestCase;
 import org.elasticsearch.action.admin.indices.shrink.ResizeType;
 import org.elasticsearch.action.admin.indices.stats.IndicesStatsResponse;
 import org.elasticsearch.action.index.IndexRequestBuilder;

--- a/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageIT.java
+++ b/x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageIT.java
@@ -44,7 +44,6 @@ import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.notNullValue;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/94935")
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class ReactiveStorageIT extends AutoscalingStorageIntegTestCase {
 

--- a/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/allocation/SearchableSnapshotDiskThresholdIntegTests.java
+++ b/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/allocation/SearchableSnapshotDiskThresholdIntegTests.java
@@ -7,7 +7,6 @@
 
 package org.elasticsearch.xpack.searchablesnapshots.allocation;
 
-import org.apache.lucene.tests.util.LuceneTestCase;
 import org.elasticsearch.action.ActionListener;
 import org.elasticsearch.blobcache.shared.SharedBlobCacheService;
 import org.elasticsearch.cluster.ClusterInfoService;

--- a/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/allocation/SearchableSnapshotDiskThresholdIntegTests.java
+++ b/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/allocation/SearchableSnapshotDiskThresholdIntegTests.java
@@ -71,7 +71,6 @@ import static org.elasticsearch.xpack.core.searchablesnapshots.MountSearchableSn
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/94935")
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class SearchableSnapshotDiskThresholdIntegTests extends DiskUsageIntegTestCase {
 

